table/tables/index.go

```diff
@@ -402,6 +402,46 @@ func (c *index) Delete(sc *stmtctx.StatementContext, txn kv.Transaction, indexed
 	return nil
 }
 
+func (c *index) GenIndexKVIter(sc *stmtctx.StatementContext, indexedValue []types.Datum, h kv.Handle, handleRestoreData []types.Datum, untouched bool) table.IndexIter {
```

Comment:
Do we need `untouched` param? Seems it not be used here.

Reply:
Ah, I will remove it